src/test/ui/rfcs/rfc-2396-target_feature-11/check-pass.rs

```diff
@@ -0,0 +1,49 @@
+// Tests the new rules added by RFC 2396, including:
+// - applying `#[target_feature]` to safe functions is allowed
+// - calling functions with `#[target_feature]` is allowed in
+//   functions which have (at least) the same features
+// - calling functions with `#[target_feature]` is allowed in
+//   unsafe contexts
+// - functions with `#[target_feature]` can coerce to unsafe fn pointers
+
+// check-pass
```

Comment:
PR checks only test on x86(_64) builders, right? I think this file needs an `only-x86_64` annotation to avoid errors about unknown target features on other builders.

The same probably applies to all other tests.